samples/index.js

```diff
@@ -0,0 +1,27 @@
+const Bynder = require('../dist/bynder-js-sdk.cjs');
+
+const bynder = new Bynder({
+  baseURL: 'https://portal.getbynder.com/api/',
+  clientId: '<your OAuth2 client id>',
+  clientSecret: '<your OAuth2 client secret>',
+  redirectUri: '<url where user will be redirected after authenticating>'
+});
+
+// Authorization URL
+bynder.makeAuthorizationURL();
+
+// Authorization token
+bynder.getToken('code');
+
+bynder
+  .getMediaList({
+    type: 'image',
+    limit: 9,
+    page: 1
+  })
+  .then(data => { // eslint-disable-line no-unused-vars
+    // TODO Handle data
+  })
+  .catch(error => { // eslint-disable-line no-unused-vars
+    // TODO Handle the error
+  });
```

Comment:
So all samples were removed and replaced with this?

Reply:
All previous samples were browser specific. Since we planned to remove browser support, this is a Node only. I agree it could be expanded tho.

Reply:
Yeah but we'll soon work on the samples anyway so fine for now I'd say.